roles/platform/tasks/install-platform-from-repo.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2024, Itential, Inc
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- name: Download the IAP file from URL
+  ansible.builtin.get_url:
+    url: "{{ iap_archive_download_url }}"
+    dest: "{{ iap_install_dir }}/"
+    mode: '0755'
+    headers: >-
+      {%- if repository_api_key is defined and iap_archive_download_url is search("jfrog") -%}
+        {"X-JFrog-Art-Api": "{{ repository_api_key }}", "Accept": "application/octet-stream"}
+      {%- else -%}
+        {"Accept": "application/octet-stream"}
+      {%- endif -%}
```

Comment:
This probably needs the same comment as the other time we set these headers.  I assume that the else condition is for a Nexus request, but I am assuming.